index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <link rel="stylesheet" href="style.css">
+        <title>first mobile app</title>
+    </head>
+
+    <body>
+        <section class="main-page">
+            <header class="main-header">
+                <nav>
+                    <tag class="header-text">John Doe</tag>
+                </nav>
+                <nav class="union">
+                    <div class="vector"></div>
+                    <div class="vector"></div>
+                    <div class="vector"></div>
+                </nav>
```

Comment:
- [ ] As the previous reviewer said, the "Jhon Doe" Represents a logo, so it would be better if you wrap it with an `<a>` tag (which represents a link) because, by convention, when a user clicks on the logo, he expects to be redirected to the main page. Take in mind that `<tag>` is not a real HTML tag, please consider changing this! 🤓 🛠️ 

- [ ] **[OPTIONAL]:** Also, instead of using two `<nav>` elements here, I suggest you use just one that wraps both elements:

```suggestion
                <nav>
                    <a href="#" class="header-text">John Doe</a>
                    <div class="union">
                        <div class="vector"></div>
                        <div class="vector"></div>
                        <div class="vector"></div>
                    </div>
                </nav>
```

Reply:
done